README.md

```diff
@@ -278,7 +275,7 @@ appropriate.
 
 ```scss
 // ./src/apps/my-new-application/my-new-application.scss
-.dpl-warm {
+.warm {
```

Comment:
Doesn't the removal of `dpl-` contradict with what's stated in [`code_guidelines.md` line 94](https://github.com/reload/dpl-react/blob/docs-typescript/docs/code_guidelines.md?plain=1#L94)?

Reply:
It does. I have reverted this change in [e4f8186](https://github.com/danskernesdigitalebibliotek/dpl-react/pull/78/commits/e4f81862eee59c08b6a037e9744dee68762e90ef).